FindMe/Parse Manager/DatabaseManager.h

```diff
@@ -20,9 +20,10 @@ NS_ASSUME_NONNULL_BEGIN
 + (void)logOutUser:(void(^)(NSError * error))completion;
 + (void)logInUser:(NSString *)usernane withPassword:(NSString *)password withCompletion:(void(^)(NSError * error))completion;
 + (void)uploadContacts:(NSArray *)contacts withCompletion:(void(^)(NSError *error))completion;
-+ (Contact *)getContactFromPFObject: (PFObject *)contactObject;
++ (void)contactFromPFObject:(PFObject *)contactObject withCompletion:(void(^)(Contact* contact))completion;
```

Comment:
funny enough, now the right name is probably `getContactFromPFObject:withCompletion:`